zebra-script/src/lib.rs

```diff
@@ -87,9 +100,26 @@ impl CachedFfiTransaction {
             .expect("serialized transaction lengths are much less than u32::MAX");
         let mut err = 0;
 
-        // SAFETY: the `tx_to_*` fields are created from a valid Rust `Vec`.
+        let serialized_all_previous_outputs = all_previous_outputs
```

Comment:
Can we rename this buffer, ptr, and len to `all_previous_outputs_serialized*`, to make it clear that it isn't a shared reference to any `self.all_previous_outputs` data?

(Putting the `serialized` last makes autocomplete a bit easier to use.)